docs/Options.md

```diff
@@ -523,6 +523,7 @@ This section show only when there are active jobs in the background.
 | `SPACESHIP_JOBS_SUFFIX` | ` ` | Suffix after the jobs indicator |
 | `SPACESHIP_JOBS_SYMBOL` | `✦` | Character to be shown when jobs are hiding |
 | `SPACESHIP_JOBS_COLOR` | `blue` | Color of background jobs section |
+| `SPACESHIP_JOBS_AMOUNT_THRESHOLD` | `1` | Number of jobs which job section will be shown |
```

Comment:
Wording isn't proper. `jobs` section will be displayed if there are any background jobs, It's only the count we're hiding. May be something like "Number of jobs _after_ which job _count_ will be displayed/shown" ?

@maximbaz might be able to provide better options.